IRremoteESP8266.cpp

```diff
@@ -450,6 +450,45 @@ void ICACHE_FLASH_ATTR IRsend::sendDenon (unsigned long data,  int nbits) {
   ledOff();
 }
 
+// Gree, from https://github.com/ToniA/arduino-heatpumpir/blob/master/GreeHeatpumpIR.cpp
+void ICACHE_FLASH_ATTR IRsend::sendGree(unsigned char data[]) {
+
+  uint8_t i = 0;
+
+  // Set IR carrier frequency
+  enableIROut(38);
+
+  // Header #1
+  mark(GREE_HDR_MARK);
+  space(GREE_HDR_SPACE);
+
+  // Data #1
+  for (; i < 4; i++)
+    sendData(GREE_BIT_MARK, GREE_ONE_SPACE, GREE_BIT_MARK, GREE_ZERO_SPACE,
+             data[i], 8, false);
+
+  // Footer #2 (010)
+  mark(GREE_BIT_MARK);
```

Comment:
You could replace this Footer section with:
`sendData(GREE_BIT_MARK, GREE_ONE_SPACE, GREE_BIT_MARK, GREE_ZERO_SPACE, 0x2, 3);`